src/main/java/net/glowstone/entity/GlowLivingEntity.java

```diff
@@ -806,6 +816,15 @@ public void damage(double amount, Entity source, DamageCause cause) {
         // apply damage
         amount = event.getFinalDamage();
         lastDamage = amount;
+
+        if (isPlayerHit(source)) {
+            playerDamageTick = ticksLived;
+        }
+
+        if (health - amount <= 0) {
```

Comment:
Should this if-statement be inside of the `isPlayerHit` if-block above?

Reply:
Strictly saying it doesn't matter, they both determine the source. But put it inside does make more sense.

Reply:
Put it inside. It'll save a few instructions when `!isPlayerHit(source)`